generic-templates/simple-snippet.html

```diff
@@ -10,6 +10,7 @@
   @icon_url: text - URL linked from icon and whole snippet when snippet is clickable. (optional / required when snippet is clickable)
   @link_logo: checkbox - Check to make replacement logo clickable.
   @replacement_logo: image - Optional. Replace logo.
+  @snippet_button_text: small text - Optional. Text for a button that links to icon_url.
```

Comment:
I realized that this is more descriptive if we rename it to `button_label`. Also please change text to 
`Optional. Text for a button next to main snippet text that links to icon_url.`